mitreattack/attackToExcel/attackToExcel.py

```diff
@@ -95,13 +96,24 @@ def write_excel(dataframes, domain, version=None, outputDir="."):
             for matrix in dataframes[objType]:  # some domains have multiple matrices
                 # name them accordingly if there are multiple
                 sheetname = "matrix" if len(dataframes[objType]) == 1 else matrix["name"] + " matrix"
-                matrix["matrix"].to_excel(master_writer, sheet_name=sheetname,
-                                          index=False)  # write unformatted matrix data to master file
+                listing = []
+                if 'ATT&CK' in matrix['name']:  # avoid printing subtype matrices to the master file
+                    matrix["matrix"].to_excel(master_writer, sheet_name=sheetname,
+                                              index=False)  # write unformatted matrix data to master file
+                    listing.append(master_writer)
```

Comment:
The master writer only gets a matrix if it has `ATT&CK` in it, which means the Mobile master file doesn't include any matrices. Ideally it should include the two non-platform-filtered matrices (Device Access and Network Based Effects).

![image](https://user-images.githubusercontent.com/28868888/128882515-bedb27ad-ad8c-45ed-a9b1-a160ee2f79f6.png)

Reply:
I think I came up with a reasonably stable way to do this a bit cleaner.